docs/source/NEXUS.rst

```diff
@@ -1,24 +1,64 @@
 NEXUS
 =====
 
-NEXUS stands for NEXt Utility for Simulation
-*Language*: C++, Geant4; *Based*: Geant4 toolkit, divided in logical blocks
+NEXUS stands for *NEXt Utility for Simulation* and can be found on Github as `next-exp/nexus <https://github.com/next-exp/nexus>`_.
 
-This section is focused on reviewing how the code is structured (:ref:`NEXUScode`)
-and its installation (:ref:`NEXUSinstallation`).
+**Language**: C++, Geant4; **Based**: Geant4 toolkit, divided in logical blocks.
 
+This section is focused on reviewing how the code is structured and where the installation guides can be found.
 
+.. note::
+  The main NEXUS documentation can be found on the `Wiki <https://github.com/next-exp/nexus/wiki>`_  of its Github repository.
 
 .. _NEXUScode:
 
 Code
 ------------
 
-Info from code
+NEXUS code is `structured <https://github.com/next-exp/nexus/wiki>`_  in a set of common tools (basic geometries, physics lists, materials database, etc)
+to ensure data consistency to be easily extended to include new geometries
+and configurations. NEXUS is written in C++, and depends on the following third-party libraries:
+
+ * **Geant4**: NEXT simulations are based on Geant4;
+ * **ROOT**: used for I/O and histogramming;
+ * **gsl**: NEXUS depends on the GNU Scientific Library for the simulation of double beta decay events;
+ * **hdf5**: output files are writen as hdf5.
+
+Important information about NEXUS code:
+ * `User-guide <https://github.com/next-exp/nexus/wiki/User-guide>`_ can be found on the NEXUS Wiki
+ * A summary of the `output format <https://github.com/next-exp/nexus/wiki/Output-format>`_ can also be found there
+ * Examples of output format are uploaded on doc-db in `NEXT-doc-1313-v1 <https://next.ific.uv.es/cgi-bin/DocDB/private/ShowDocument?docid=1313>`_
+
+Tags
+------------
+A list of the different NEXUS `tags <https://github.com/next-exp/nexus/wiki/Tags>`_ can be found on the Wiki.
+Latest NEXUS tag is >= v7_00_00, with the following recommended versions:
```

Comment:
I would say: "Starting from nexus tag v7_00_00, Geant4 version >= 11.0 is needed".
The ROOT version in principle can be whatever, except if you use CMake for the building, in which case it must be >= 6.23 and different from 6.24.6 😅. I don't know how to explain this easily.

Reply:
Perhaps the simplest approach is demanding a ROOT version >=6.25. I can modify the CMake script to enforce it.

Reply:
I have added:

```
Starting from nexus tag v7_00_00, the following versions are needed:
 * Geant4 version >= 4.11.0.
 * ROOT >= 6.26.
```
 
Does it sound good to you?